### DISCUSSION TITLE
How to docker-build saleor as prod mode?

### DISCUSSION THREAD

**Discussion by groovejumat** [saleor/saleor#6142] [Deployment]
I want to use saleor prod-mode test. dev-mode docker authentication system doesn't get jwt cookie. So I want to build saleor prod mode and test about authentication for our reference.

Can I get some reference for this work? Please I really need this.

**Comment by karolkielecki** [comment]
Hey @groovejumat! Is the issue solved? Do you have more questions regarding this topic? 

For those who are looking for more information regarding jwt, please check our unreleased docs for 2.11: https://docs.saleor.io/docs/next/developer/users/#creating-a-jwt-access-token
